scripts/tests/test_phase44c_projection_root_probe.py

```
@@ -0,0 +1,74 @@
+
+import importlib.util
+import json
+import pathlib
+import tempfile
+import unittest
+
+
+ROOT = pathlib.Path(__file__).resolve().parents[2]
+CHECKER = ROOT / "scripts" / "check_phase44c_projection_root_probe.py"
+MANIFEST = ROOT / "docs" / "engineering" / "design" / "phase44c-projection-root-manifest.json"
+STWO_ROOT = pathlib.Path("/tmp/zkai-research/repos/stwo")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/zkai-research/repos/stwo"

(S108)